src/ui/components/Library/Team/View/Tests/Overview/TestOverviewContent.tsx

```diff
@@ -0,0 +1,77 @@
+import { useContext } from "react";
+
+import { FailureRates } from "shared/test-suites/TestRun";
+
+import { TestContext } from "../TestContextRoot";
+import { TestErrorList } from "./TestErrorList";
+import styles from "../../../../Library.module.css";
+
+export function TestOverviewContent() {
+  const { filterByText, testId, tests } = useContext(TestContext);
+  const hasFilters = filterByText !== "";
+  const test = tests.find(test => test.testId === testId);
+
+  let children = null;
+
+  if (test) {
+    if (!hasFilters || tests.find(test => test.testId === testId)) {
+      children = (
+        <div className="flex flex-col" data-test-id="TestRunSummary">
+          <div className="flex flex-row items-center justify-between gap-1 border-b border-themeBorder py-2 px-4">
+            <div className="overflow-hidden overflow-ellipsis whitespace-nowrap text-lg font-medium">
+              {test.title}
+            </div>
+          </div>
+          <Stats failureRates={test.failureRates} />
+          <TestErrorList errorFrequency={test.errorFrequency} executions={test.executions} />
+        </div>
+      );
+    }
+  } else {
+    children = (
+      <div className="flex h-full items-center justify-center p-2">
+        <div className=" rounded-md bg-chrome py-2 px-3 text-center">
+          Select a test to see its details here
+        </div>
+      </div>
+    );
+  }
+
+  return (
+    <div className={`flex h-full flex-col text-sm transition ${styles.runOverview} `}>
+      {children}
+    </div>
+  );
+}
+
+function Stats({ failureRates }: { failureRates: FailureRates }) {
+  const { hour, day, week, month } = failureRates;
+
+  return (
+    <div className="border-b border-themeBorder py-2 px-4">
+      <div>
```

Comment:
Pretty sure this needs to be overflow as well
```suggestion
<div class="overflow-y-auto">
```